### DISCUSSION TITLE
Air Quality in Header

### DISCUSSION THREAD

**Discussion by Coehill** [breezy-weather/breezy-weather#1146] [Ideas]
### Describe your idea

From Settings > Main Screen > Details in header, there are several options, but it does not include air quality.  It woild be helpful to include air quality here so that we could see it at a glance when opening the app.

I recently started using Breezy and have really been liking it. Thank you to the developers who have done such a great job on this!

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Current overall air quality index (value) with description (text)?

**Reply by Coehill** [reply]
That would be perfect!

**Comment by papjul** [comment]
A new homepage is in progress (#1928), and the details on top will be removed in favor of Material 3 Expressive blocks.
You will be able to move the new half-sized Air quality block on top, or wait for #1899 to get the info on top as a small text, if you prefer.
Closing as the feature will no longer exist
